### PR TITLE
Add default accounts for admin, agency worker, and general users

### PR DESCRIPTION
**Pull request by rileynwong**

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -65,18 +65,38 @@ def add_fake_data(number_users, number_reports):
     User.generate_fake(count=number_users)
     IncidentReport.generate_fake(count=number_reports)
 
+
```
Comment (sandlerben) [inline manage.py:61]: Why the added space here? Was it not there before?

File: manage.py [hunk 2]
```diff
@@ -65,18 +65,38 @@ def add_fake_data(number_users, number_reports):
...
 @manager.command
 def setup_dev():
     """Runs the set-up needed for local development."""
     setup_general()
 
     # Create a default admin user
-    admin = User(email='user@example.com',
+    admin = User(email='user@admin.com',
```
Comment (sandlerben) [inline manage.py]: Why don't we make the emails like `admin@user.com`?

**Comment by sandlerben** [comment]
:shipit:

File: manage.py (hunk 1 above)
Comment (rileynwong) [inline manage.py:61]: yea pep8 was >:( at me

Comment (sandlerben) [inline manage.py:61]: Yay pep8

**Pull request closed without merging**